plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -57,8 +58,19 @@ object ScalafmtPlugin extends AutoPlugin {
         "(By default this means the Compile and Test configurations.)"
     )
   }
+
   import autoImport._
 
+  case class ScalafmtAnalysis(failedScalafmtCheck: List[File])
```

Comment:
Is the choice of `List` driven by sjsonnew? It looks like it's always handled as a `Set` internally.